www/index.php

```diff
@@ -495,7 +495,8 @@
 					<div id="unitsQuestionMark" class="questionMark"></div>
 					<label id="selectUnitsControl">Units
 						<select id="selectUnits" data-dojo-type="dijit/form/Select">
-							<option value='defaultSelect'>No Units</option>
+							<option value='defaultSelect'>--Select--</option>
+							<option value='no-units'>No Units</option>
```

Comment:
This is causing "no-units" to get rendered on the node.  If there's no units, we should just display nothing in that spot.  Perhaps we can just make the value null or the empty string.